src/backend/c-lisp.py

```diff
@@ -284,6 +404,14 @@ def gen_expr(self, expr, res_sym=None):
             return self.gen_var_expr(expr, res_sym)
         elif self.is_binary_expr(expr):
             return self.gen_binary_expr(expr, res_sym)
+        elif self.is_ptradd_expr(expr):
```

Comment:
is ptradd etc too different from fadd etc instructions?

Reply:
The main difference is this: the other expressions ( e.g. `fadd`, `add`) have produces values of a fixed type (`float` and `int` in this example). For this reason, even temporary symbols created by `ptradd` instructions need to be tracked, unlike with the others.

For example, in the expression `(load (ptradd arr 5))`,
* `ptradd` produces a temporary symbol
* `load` emits a Brilisp `load` that loads from the temporary symbol. The type of the resulting value, the value that is being loaded, cannot be determined unless the temporary symbol from `ptradd` is tracked.

Reply:
This is also why we need self.pointer_types